test/parallel/test-assert.js

```diff
@@ -533,11 +551,18 @@ a.throws(makeBlock(thrower, TypeError), (err) => {
   const noop = () => {};
   assert.throws(
     () => { a.throws((noop)); },
-    common.expectsError({
-      code: 'ERR_ASSERTION',
-      message: /^Missing expected exception\.$/,
-      operator: 'throws'
-    }));
+    e => {
+      assert.strictEqual(e.code, 'ERR_ASSERTION');
+      assert(/^Missing expected exception\.$/.test(e.message));
+      assert.strictEqual(e.operator, 'throws');
+      // stack shouldn't start with assert's innerFail, but caller instead
+      const lines = e.stack.split('\n')
+      assert.equal(lines.length, 11)
+      assert.ok(lines[0].includes(e.message))
+      assert.ok(lines[1].includes('at assert.throws'))
```

Comment:
Hm, I am not certain if that is actually true? I thought that frame would actually be excluded? Do the tests pass?